Cloudinary/Classes/Core/Features/Helpers/CLDTransformation.swift

```diff
@@ -1472,12 +1472,17 @@ import CoreGraphics
      - parameter videoCodec:        The video codec to set.
      - parameter videoProfile:      The video profile to set.
      - parameter level:             The level to set.
+     - parameter bframes:           Should use B-frames.
      
      - returns:                     The same instance of CLDTransformation.
      */
     @discardableResult
-    open func setVideoCodecAndProfileAndLevel(_ videoCodec: String, videoProfile: String, level: String? = nil) -> Self {
-        return level == nil ? setVideoCodec("\(videoCodec):\(videoProfile)") : setVideoCodec("\(videoCodec):\(videoProfile):\(level!)")
+    open func setVideoCodecAndProfileAndLevelAndBFrames(_ videoCodec: String, videoProfile: String, level: String? = nil, bframes: Bool? = nil) -> Self {
```

Comment:
Changing the function name will break the SDK for existing users
There are 2 ways you can approach this:
1. Just add the bframe as optional value
2. overload the function and add the bframes variable.

Reply:
Overloading seems the logical solution, good catch.